reference/v8.json

```diff
@@ -163,7 +163,9 @@
     "buffer": {
       "type": "number",
       "default": 128,
-      "doc": "Tile buffer size on each side (measured in 1/512ths of a tile; higher means fewer rendering artifacts near tile edges but slower performance)."
+      "maximum": 512,
```

Comment:
There are no sources that accept both `tileSize` and `buffer` options. 

> Size of the tile buffer on each side. A value of 0 produces no buffer. A value of 512 produces a buffer as wide as the tile itself. Larger values produce fewer rendering artifacts near tile edges and slower performance.

I worded this documentation to avoid the need to explain tile resizing. The units are, for practical purposes, opaque and are explained as such.